notebooks/using-mlflow/no-code-deployment/deploying_with_mlflow.ipynb

```diff
@@ -0,0 +1,469 @@
+    "\n",
+    "\n",
+    "| Feature | ACI | AKS | MIR |\n",
+    "| :- | :-: | :-: | :-: |\n",
+    "| Deploying models logged with MLflow to real time inference | ☑️* | ☑️* | ☑️* |\n",
+    "| Deploying models logged with MLflow to batch inference | ☐** | ☐** | ☑️ |\n",
+    "| Deploying models logged with MLflow locally | ☑️ | ☑️ | ☐*** |\n",
+    "| Deploying models with ColSpec signatures | ☑️**** | ☑️**** | ☑️**** |\n",
+    "| Deploying models with TensorSpec signatures | ☑️ | ☑️ | ☑️ |\n",
+    "| Debug online endpoints locally in Visual Studio Code (preview) | ☐ | ☐ | ☐ |\n",
+    "\n",
+    "**Notes:**\n",
+    "- (*) Spark flavor is not supported at the moment.\n",
+    "- (**) We suggest you to use Azure Machine Learning Pipelines with Parallel Run Step.\n",
+    "- (***) For deploying MLflow models locally, use the command `mlflow models serve -m <MODEL_NAME>`. Configure the environment variable `MLFLOW_TRACKING_URI` with the URL of your tracking server.\n",
+    "- (****) Data type `mlflow.types.DataType.Binary` is not supported as column type. For models that works with images, we suggest you to use Base64 encoding schemes with a `mlflow.types.DataType.String` column type, which is commonly used when there is a need to encode binary data that needs be stored and transferred over media. \n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3a20f351",
+   "metadata": {},
+   "source": [
+    "There are two workflows for deploying models to Azure ML:\n",
+    "\n",
+    "- Using the MLflow plugin [azureml-mlflow](https://pypi.org/project/azureml-mlflow/).\n",
+    "- Using Azure ML CLI/SDK v2.\n",
+    "\n",
+    "If you are familiar with MLflow are you wish to continue using the same set of methods, keep using the approach. If, on the other hand, you are more familiar with the Azure ML CLI, you want to automate deployments using CI/CD pipelines, or you want to keep deployments configuration in a git repository, we recommend you to use the Azure ML CLI v2."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "89167b93",
+   "metadata": {},
+   "source": [
+    "## Deploying models using Azure ML plugin for MLflow\n",
+    "\n",
+    "The MLflow plugin azureml-mlflow can deploy models to Azure ML, either to Azure Kubernetes Service (AKS), Azure Container Instances (ACI) and Managed Inference Service (MIR) for real-time serving.\n",
+    "\n",
+    "> Note that deploying to Managed Inference Service - Batch endpoints is not supported in the MLflow plugin at the moment.\n",
+    "\n",
+    "### Prerequisites\n",
+    "\n",
+    "- Install the `azureml-mlflow` package.\n",
+    "- Configure the MLflow tracking URI or MLflow's registry URI to point to the workspace you are working on. See [MLflow Tracking URI to connect with Azure Machine Learning](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-use-mlflow) for more details.\n",
```

Comment:
This points to v1 SDK should we have another doc for v2 and point it to that ?

Reply:
I will update that documentation, but the link should be the same.